low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -369,14 +380,13 @@ def process(self):
 
 
 class ResetCmd(Cmd):
-    """Performs SSPL config reset. Options: hard, soft.
-    'hard' is used to reset configs and clean log directory where
-    'soft' is to clean only the data path.
+
+    """Performs SSPL config reset.
+    Reset Interface is used to reset configs and clean log directory
```

Comment:
Config is not touched in reset now right.

>Reset Interface is used to reset meta data and clean log directory

Reply:
Done the change